### DISCUSSION TITLE
Global and Relation Search cancel (X) right padding not the same as left padding on magnifying glass

### DISCUSSION THREAD

**Discussion by RhysLees** [laravel/nova-issues#4963] [Ideas & Feature Requests]
- Laravel Version: 9.35.1
- Nova Version: 4.16.1
- PHP Version: 8.0 | 8.1
- Database Driver & Version: Mysql
- Operating System and Version: Win 11 WSL2 
- Browser type and version: Chrome Latest
- Reproduction Repository: https://github.com/RhysLees/NovaIssues/

#### Global Search
![image](https://user-images.githubusercontent.com/43909932/197239416-5d05414a-0976-4568-910b-8d72e7301fc1.png)

#### Relation Search
![image](https://user-images.githubusercontent.com/43909932/197239509-519637e0-8558-48af-9f46-30fb07a8e2a2.png)

**Comment by crynobone** [comment]
The `x` is provided by some browser, and generally we don't mess with browser's default behaviour.

**Comment by RhysLees** [comment]
> The `x` is provided by some browser, and generally we don't mess with browser's default behaviour.

Fair, just seems out of place also the style of it is not the same so it looks out of place in general

**Comment by piljac1** [comment]
I agree that it doesn't look right. Could hiding the default "x" via CSS and adding a custom one that is more in-line with Nova's style be considered ?
